src/general_manager/bucket/databaseBucket.py

```
@@ -1,11 +1,5 @@
 from __future__ import annotations
-from typing import (
-    Type,
-    Any,
-    Generator,
-    TypeVar,
-    TYPE_CHECKING,
-)
+from typing import Type, Any, Generator, TypeVar, TYPE_CHECKING, cast
```

Ruff (0.11.9)

`typing.cast` imported but unused

Remove unused import: `typing.cast`

(F401)